src/lib/mutate-processor.ts

```diff
@@ -79,7 +87,8 @@ export async function mutateProcessor(
         // Run the action
         await action.mutateCallback(wrapped);
 
-        Log.info(actionMetadata, `Mutation action succeeded (${label})`);
+        // Log the alias on success
+        aliasLogger.info(actionMetadata, `Mutation action succeeded (${label})`);
```

Comment:
```suggestion
        Log.info(actionMetadata, `Mutation action succeeded (${label})`);
```

Where did we land on discussion whether or not the childLogger should log controller stuff? I think we may want to keep that responsibility on Pepr but it is up for debate.

Reply:
also, even if we decide to delegate controller logging within the action to the child logger, shouldn't this be conditional? Not sure what the repercussions of logging with the child logger if it is not being used in the action

Reply:
Updated to use Log